### DISCUSSION TITLE
migrazione rasp4 to miniPC

### DISCUSSION THREAD

**Discussion by kimuss** [indomus/forum#3004] [GENERALE - Discussioni generiche]
salve, vorrei chiedere un consiglio riguardo la migrazione che sto affrontando da un rasp4 a un mini pc.
Su rasp ho tutto sotto docker, core di HA, pihole, caddy, vpn e altre cosette, ovviamente il 90% è frutto delle guide che seguo costamente su indomus e dove non farò mai abbastanza per ringraziarvi.
Detto questo ho letto molto chiaramente sul sito, il vostro disappunto nella creazione di un'istanza HA sotto proxmox, in qualunque salsa, tant' è che esplicitamente non realizzate guide al riguardo.
Trovandomi pressochè d' accordo con il flusso di pensiero espresso in svariate situazioni espresse dal team di indomus, vorrei chiedere quale procedura consigliate per poter riprodurre il sistema mio attuale aggiungendo features che a me interessano, come per esempio avere nel mini pc, pfsense, o truenas o altre situazioni che non possono stare sotto docker.
Mi piacerebbe avere un consiglio, qualora aveste tempo e voglia di farlo, su come approccereste a strutturare il tutto.
grazie

**Comment by trebestie** [comment]
Incus o proxmox

**Reply by KC-inDomus** [reply]
certo, a uno che vuol migrare da uno stack docker gli consigli incus o proxmox.

**Reply by trebestie** [reply]
Cosa impedisce di usare docker in un container incus o lxc?

**Reply by KC-inDomus** [reply]
nulla: ma se già è pratico di docker, perché dovrebbe cambiare approccio?

**Reply by trebestie** [reply]
l'OP chiedeva come mettere insieme anche cose che non possono stare sotto docker. 
A me la suddivisione in container sembra la più adatta,

**Reply by kimuss** [reply]
mm incus non conosco, mi informerò
grazie

**Comment by KC-inDomus** [comment]
> salve, vorrei chiedere un consiglio riguardo la migrazione che sto affrontando da un rasp4 a un mini pc. Su rasp ho tutto sotto docker, core di HA, pihole, caddy, vpn e altre cosette, ovviamente il 90% è frutto delle guide che seguo costamente su indomus e dove non farò mai abbastanza per ringraziarvi. Detto questo ho letto molto chiaramente sul sito, il vostro disappunto nella creazione di un'istanza HA sotto proxmox, in qualunque salsa, tant' è che esplicitamente non realizzate guide al riguardo. Trovandomi pressochè d' accordo con il flusso di pensiero espresso in svariate situazioni espresse dal team di indomus, vorrei chiedere quale procedura consigliate per poter riprodurre il sistema mio attuale aggiungendo features che a me interessano, come per esempio avere nel mini pc, pfsense, o truenas o altre situazioni che non possono stare sotto docker. Mi piacerebbe avere un consiglio, qualora aveste tempo e voglia di farlo, su come approccereste a strutturare il tutto. grazie

banalmente: tira su debian, docker, e continua com'hai fatto ottimamente sin qui. lascia perdere proxmox.

**Comment by kimuss** [comment]
ok, quindi lascio stare del tutto programmi come pfsense o truenas (scale)?
